benchmarks/src/search.cpp

```diff
@@ -32,7 +32,8 @@ template <size_t Size, bool Last_is_different>
 constexpr std::array fill_pattern_array = make_fill_pattern_array<Size, Last_is_different>();
 
 template <size_t Size, bool Last_is_different>
-constexpr std::string_view fill_pattern_view = fill_pattern_array<Size, Last_is_different>;
+constexpr std::string_view fill_pattern_view{
+    fill_pattern_array<Size, Last_is_different>.data(), fill_pattern_array<Size, Last_is_different>.size()};

```

Comment:
Did not want to construct `string_view` from `array` directly, this was error, not warning